src/com/gfredericks/test/chuck/generators.cljc

```diff
@@ -144,7 +149,8 @@
   collection, in the same order. For collections of distinct elements
   this is effectively a subset generator, with an ordering guarantee."
   [elements]
-  (for [bools (apply gen/tuple (repeat (count elements) gen/boolean))]
+  (for [bools (apply gen/tuple
+                     (repeat (count elements) gen/boolean))]
```

Comment:
I won't be a stickler for this, but I'd prefer to avoid whitespace changes; don't mind a separate PR for just whitespace though.